janusgraph-hadoop/src/test/java/org/janusgraph/hadoop/ConfigurationTest.java

```diff
@@ -0,0 +1,29 @@
+package org.janusgraph.hadoop;
```

Comment:
You have to add the copyright header.
I would move the test into following class: janusgraph/janusgraph-test/src/test/java/org/janusgraph/diskstorage/configuration/ConfigurationTest.java

Reply:
Hm.. 'janusgraph-test' project has no hadoop dependency.
Also 'JanusGraphHadoopSetupImpl' uses 'ModifiableHadoopConfiguration' instead of 'ModifiableConfiguration' or 'BasicConfiguration'.
So I added test code to 'janusgraph-hadoop'.
Anyway do I modify test code using 'ModifiableConfiguration' or 'BasicConfiguration'?

Reply:
I'm okay with the current implementation. I thought it is independent from Hadoop because there are no changes to ModifiableHadoopConfiguration which extends ModifiableConfiguration. I think this part of the test should in the core test otherwise every change in configuration part has to be tested with Hadoop from now.